base/person.rb

```diff
@@ -1,16 +1,17 @@
-require_relative 'nameable'
-require_relative 'capitalizedecorator'
-require_relative 'trimmerdecorator'
+require_relative '../decorator/nameable'
+require_relative '../decorator/capitalizedecorator'
+require_relative '../decorator/trimmerdecorator'
 
 class Person < Nameable
-  attr_reader :id, :name, :age
+  attr_reader :id, :name, :age, :rental
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
```

Comment:
- [ ] you are using both `@rental` and `@rentals` please make sure that after calling it `@rentals` to use this naming convention everywhere.

Reply:
well resolved! ✔️